InventoryBuilder.py

```diff
@@ -12,6 +12,7 @@
 
 class InventoryBuilder:
     def __init__(self, json):
+        self.token = self.get_token(self)
```

Comment:
please add flask entry to let token be queried

Reply:
and adapt the test case to use the token ;)

Reply:
I'm on my way